prow/scripts/provision-vm-and-start-kyma-k3d.sh

```diff
@@ -30,8 +30,12 @@ cleanup() {
   set +e
 
   #shellcheck disable=SC2088
-  utils::receive_from_vm "${ZONE}" "kyma-integration-test-${RANDOM_ID}" "~/kyma/tests/fast-integration/junit_kyma-fast-integration.xml" "${ARTIFACTS}"
-  gcloud compute instances stop --async --zone="${ZONE}" "kyma-integration-test-${RANDOM_ID}"
+  if [[ "$ISTIO_INTEGRATION_ENABLED" == "true" ]]; then
+    utils::receive_from_vm "${ZONE}" "kyma-integration-test-${RANDOM_ID}" "~/kyma/tests/components/istio/junit-report.xml" "${ARTIFACTS}"
```

Comment:
Example pipeline: https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/14464/barchw_test_of_prowjob_pre-main-kyma-integration-k3d-istio-eval/1531934724627369984

Reply:
@cnvergence I think this line will need to be fixed because we should use bdd specific reporter like: https://github.com/gkushang/cucumber-html-reporter
IMO this should be handled in the followup issue.

Reply:
Right, let's remove it in Kyma-side makefile in the followup